omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -654,257 +770,361 @@
+                                    </a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" data-toggle="tab" href="#medication">Medication</a>
+                                </li>
+                                <li class="nav-item">
+                                    <a class="nav-link" data-toggle="tab" href="#investigations">Tests/Investigations
+                                        and
+                                        DSDM
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="card-body">
+                            <div class="tab-content">
+                                <div class="tab-pane active" id="appointment-vitals-nutrition">
+                                    <section>
+                                        <div class="card">
+                                            <div class="card-header">Appointment</div>
+                                            <div class="card-body">
+                                                <div class="row">
+                                                    <div class="col-md-3" id="visit-date">
+                                                        <label>Visit Date</label>
+                                                        <encounterDate default="today"
+                                                                       disallowMultipleEncountersOnDate="block"/>
+                                                    </div>
+                                                    <div class="col-md-3">
+                                                        <excludeIf velocityTest="$patient.person.dead ==true">
+                                                            <lookup complexExpression="#set($transfer_out_date = '')"/>
+                                                            <ifMode mode="ENTER">
+                                                                <span id="transfer_out_date_id" class="hidden">
+                                                                    <lookup
+                                                                            complexExpression="#set($transfer_out_date=($fn.latestObs('99165').valueDatetime)) #set($lost_to_follow_date=$fn.latestObs('99165').valueDatetime) #set($transfer_out=$fn.latestObs('90306').valueCoded.name) $transfer_out_date"/>
+                                                                </span>
+                                                                <includeIf velocityTest="$transfer_out=='YES'">
+                                                                    <label>Return date</label>
+                                                                    <obs id="5096" conceptId="5096"
```

Comment:
@slubwama Global - put back all the ID references that have been replaced by concept IDs

Reply:
done